maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJCMojo.java

```diff
@@ -984,7 +984,29 @@ public void setPlugins(Dependency[] plugins) {
 		this.plugins = plugins;
 	}
 
-	@Component
+    /**
+     * <p>
+     * A list of artifacts to exclude from XJC Plugins resolveTransitively function.
+     * </p>
+     * <p>
+     * If left undefined, then <code>xerces:xercesImpl</code> and <code>xml-apis</code> will
+     * be excluded by default from the classpath resolution.
+     * </p>
+     */
+    @Parameter
+    private String[] artifactExcludes = new String[] {
```

Comment:
This field name should be more consistent and the format of the value should follow syntax of other plugins that do simialr. Is it comma separated artifactIds, groupIds, or a custom GAV-style format? Does it support doing a full gav+classifier?

Either we follow other conventions and do an artifactIdExclude, groupIdExcludes, or dependencyExcludes (similar gav format as to other maven plugins)

Reply:
The value is used somewhere else in the code and there, it accepts `artifactId` or `groupId:artifactId` syntax.

Configuration will be more something like :

```xml
<tag>groupid:artifactid<tag>
<tag>artifactid2<tag>
<tag>groupid2:artifactid3<tag>
```